src/nimlsppkg/suggestlib.nim

```diff
@@ -115,11 +115,13 @@ createFullCommand(def)
 createFullCommand(use)
 createFullCommand(dus)
 createFileOnlyCommand(chk)
-#createFileOnlyCommand(`mod`)
 createFileOnlyCommand(highlight)
 createFileOnlyCommand(outline)
 createFileOnlyCommand(known)
 
+proc `mod`*(nimsuggest: NimSuggest, file: string, dirtyfile = ""): seq[Suggest] =
```

Comment:
Was not sure how I could do this otherwise - will change it as you see fit.

Reply:
Eh, it's a special case so it should be fine.